pkg/bot/discord.go

```diff
@@ -10,10 +10,10 @@ import (
 	"github.com/bwmarrin/discordgo"
```

Comment:
Out of curiosity: Can we somehow disable link previews from the help message? 🤔 I don't expect it's possible with a simple message, but maybe there's is a simple way in `discordgo` 🤔 

![Screenshot 2022-09-19 at 12 14 48](https://user-images.githubusercontent.com/7155799/190996284-99846036-acd2-4313-a13b-d2abfa3458ff.png)

Reply:
Unfortunately, I don't see such option.